pkg_defs/ska_sun/meta.yaml

```diff
@@ -20,16 +20,20 @@ requirements:
     - setuptools_scm
     - setuptools_scm_git_archive
     - ska_helpers
+    - testr
   run:
     - python
     - ska.quatutil
     - chandra.time
     - quaternion
     - numpy
+    - ska_helpers
+    - testr
 
 test:
   imports:
     - Ska.Sun
+    - ska_sun
 
 about:
-  home: https://github.com/sot/Ska.Sun
+  home: https://github.com/sot/ska_sun
```

Comment:
Out of scope, but maybe https://sot.github.io/ska_sun/ would be a better `home`?  I'm pleased to see that the new flat package name just works, though of course the docs are outdated.

Reply:
And actually maybe I'm disagreeing with myself already. Perhaps the git repo is the best home instead of the docs. I don't know.

Reply:
I think https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html suggests doc_url for that, but hard to know how much we want to use the conda conveniences and how much is already coded in skare3_tools.